acceptance/suites/puppet3_tests/020_backwards_compat/external_fact_output_test.rb

```diff
@@ -2,11 +2,14 @@
 
 test_name 'executable external fact'
 
-skip_test 'Executable external facts broken until PUP-4420'
-
-studio = "/tmp/simmons-studio-#{Process.pid}"
 agent = nonmaster_agents().first
 
+# This skip should be removed when executable external facts work for Puppet 4.x
+# on Linux-based platforms again (PUP-4420)
```

Comment:
Thanks, @joshcooper.  We've intended to bump our Puppet submodule reference to 4.1 once it got tagged.  I'll make sure that, as we do that, this skip is removed.  Will do that on a separate PR.